book/src/simpleexample.md

```diff
@@ -145,3 +145,56 @@ eth_friend(?1, ?2, ?3, ?4) = and<
     Equal(?5["attestation"], ?3[?4])
 >
 ```
+
+## Another perspective
+When working with statements and operations, it might be useful to see them from another perspective:
+
+- A *statement* can be seen as the *constraints* of a traditional zk-circuit,
+  which can be true or false.
+- A *predicate* is a relation formula, which when filled with values becomes a
+  *statement*.
+- An *operation* comprises the deduction rules, which are rules used to deduce
+  new statements from previous statements or used to construct new statements
+  from values.
+
+$$
+predicate \cong circuit/relation to be fulfilled\\
+statement \cong constraints filled with the witness\\
+operations \cong deduction~rules
+$$
+
+
+For example,
+- `Equal` for integers is a predicate.
+- `5 Equal 5` is a statement (true)
+- `5 Equal 4` is a statement (false)
+
```

Comment:
This could use an example of an "operation", i.e. deduction rule.  Like transitive equality.  (Doesn't necessarily have to map 1:1 to POD2 operations to be a helpful example, but it would be good if it were.)